lib/plug/builder.ex

```diff
@@ -262,7 +262,7 @@ defmodule Plug.Builder do
 
       quote do
         require Logger
-        Logger.unquote(level)(unquote(message))
+        :ok = Logger.unquote(level)(unquote(message))
```

Comment:
@ciastek of course, feel free to make another PR :). The comment sounds good as well.